### ISSUE TITLE
Provide a way to run only a subset of scenarios

### ISSUE THREAD

**Issue by davidecavestro** [alphagov/gradle-gatling-plugin#17]
It would be handy having a way to run only a subset of tests available.
Discriminating through _.scala_ file path would be enough, i.e. 
```gradle
gatling {
  include 'groupX/**' // runs only scenarios under 'groupX' package
}
```

**Comment by alxndrsn** [comment]
Can't you do this already via:
```gatling {
    scenarios = [ ... some list of scenarios ... ]
}```
?

**Comment by davidecavestro** [comment]
ouch... not tried so far: I've seen no documentation about this aspect, and didn't figured out after a fast look to the code.
I'm going to try it ASAP, but at the moment it fails trying to download a non-snapshot artifacts from sonatype snapshots maven repo, i.e. 

>davide@LAP_DEV_03:~/git/stresstest$ ./gradlew gatlingTest
>:compileJava UP-TO-DATE
>:compileScala UP-TO-DATE
>:processResources UP-TO-DATE
>:classes UP-TO-DATE
>:jar UP-TO-DATE
>:assemble UP-TO-DATE
>:compileTestJava UP-TO-DATE
>:compileTestScala
>
>FAILURE: Build failed with an exception.
>
>* What went wrong:
>Could not resolve all dependencies for configuration ':testCompile'.
> > Could not find io.gatling.highcharts:gatling-charts-highcharts:2.1.3.
>  Searched in the following locations:
>      https://oss.sonatype.org/content/repositories/snapshots/io/gatling/highcharts/gatling-charts-highcharts/2.1.3/gatling-charts-highcharts-2.1.3.pom
>      https://oss.sonatype.org/content/repositories/snapshots/io/gatling/highcharts/gatling-charts-highcharts/2.1.3/gatling-charts-highcharts-2.1.3.jar
>  Required by:
>      :stresstest:unspecified
> > Could not find com.nimbusds:nimbus-jose-jwt:2.22.1.
>  Searched in the following locations:
>      https://oss.sonatype.org/content/repositories/snapshots/com/nimbusds/nimbus-jose-jwt/2.22.1/nimbus-jose-jwt-2.22.1.pom
>      https://oss.sonatype.org/content/repositories/snapshots/com/nimbusds/nimbus-jose-jwt/2.22.1/nimbus-jose-jwt-2.22.1.jar
>  Required by:
>      :stresstest:unspecified

I'm using gradle 2.4.

PS: I've just seen there's some enhancements you could be interested on at https://github.com/htdan/gradle-gatling-plugin-1/commits/master

**Comment by alxndrsn** [comment]
The scenario magic happens here https://github.com/alphagov/gradle-gatling-plugin/blob/master/src/main/groovy/uk/gov/gradle/plugins/GatlingPlugin.groovy#L33 via https://github.com/alphagov/gradle-gatling-plugin/blob/master/src/main/groovy/uk/gov/gradle/plugins/GatlingPluginExtension.groovy

**Comment by alxndrsn** [comment]
And ref repos, it looks like https://github.com/alphagov/gradle-gatling-plugin/blob/master/src/main/groovy/uk/gov/gradle/plugins/GatlingPlugin.groovy#L23 needs updating to point to the sonatype release repo, unless these are available in `mavenCentral()`, in which case that would be more appropriate.

I'm no longer at alphagov, but PRs are likely appreciated.

**Comment by davidecavestro** [comment]
I confirm both _gatling-charts-highcharts_ and _nimbus-jose-jwt_ are available from _mavenCentral()_, so maybe it would be safer avoid adding repositories from the plugin... i.e. someone could prefer to configure a proxy repo on the gradle project using the plugin just to avoid direct access to maven central

**Comment by davidecavestro** [comment]
I can commit and file a pull request, but I've not tested it completely cause it fails running the scenarios.. it seems like gatling now searches for a _target/test-classes_ folder, while scenario classes gets compiled at _build/classes/test_
>:gatlingTest
>Executing gatling scenarios: [my.FooScenario]
>Exception in thread "main" java.nio.file.NoSuchFileException: target/test-classes
>        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
>        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
>        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
>        at sun.nio.fs.UnixFileSystemProvider.newDirectoryStream(UnixFileSystemProvider.java:427)
>        at java.nio.file.Files.newDirectoryStream(Files.java:457)
>        at io.gatling.core.util.PathHelper$RichPath$.deepListAux$1(PathHelper.scala:99)
>        at io.gatling.core.util.PathHelper$RichPath$.deepList$extension(PathHelper.scala:105)
>        at io.gatling.core.util.PathHelper$RichPath$.deepFiles$extension(PathHelper.scala)
>        at io.gatling.app.classloader.SimulationClassLoader.simulationClasses(SimulationClassLoader.scala:55)
>        at io.gatling.app.Gatling.loadSimulations(Gatling.scala:92)
>        at io.gatling.app.Gatling.start(Gatling.scala:70)
>        at io.gatling.app.Gatling$.fromArgs(Gatling.scala:59)
>        at io.gatling.app.Gatling$.main(Gatling.scala:44)
>        at io.gatling.app.Gatling.main(Gatling.scala)
>:gatlingTest FAILED

**Comment by davidecavestro** [comment]
So:
 1. about my original issue, ```gatling { scenarios = [ 'foo.FooScenario' ]}``` works like a charm! (maybe the README could mention it)
 2. about other issues, I've fixed them at #18
